src/virtual-machine.js

```diff
@@ -1071,6 +1071,38 @@ class VirtualMachine extends EventEmitter {
             this.editingTarget.postSpriteInfo(data);
         }
     }
+
+    /**
+     * Reorder the costumes of a target if it exists. Return whether it succeeded.
+     * @param {!string} targetId ID of the target which owns the costumes.
+     * @param {!number} costumeIndex index of the costume to move.
+     * @param {!number} newIndex index that the costume should be moved to.
+     * @returns {boolean} whether the target was found for reordering.
+     */
+    reorderCostume (targetId, costumeIndex, newIndex) {
+        const target = this.runtime.getTargetById(targetId);
+        if (target) {
+            target.reorderCostume(costumeIndex, newIndex);
+            return true;
```

Comment:
Should this return what `target.reorderCostume` returns? (If that is the case, the comment above should probably also be updated to say that this function returns 'whether a reorder occurred').